examples/inverse/plot_morph_VolSourceEstimate.py

```diff
@@ -0,0 +1,282 @@
+    morph['mapping'] = imwarp.DiffeomorphicMap(None, [])
+    morph['mapping'].__dict__ = morph_in.get('mapping')
+    morph['affine'] = imaffine.AffineMap(None)
+    morph['affine'].__dict__ = morph_in.get('affine')
+
+    morph['affine_reg'] = morph_in.get('affine_reg')
+    morph['domain_shape'] = morph_in.get('domain_shape')
+
+    return morph
+
+
+###############################################################################
+# Execute example
+
+# Settings
+voxel_size = (5., 5., 5.)  # of the destination volume
+
+# Setup path
+data_path = sample.data_path()
+
+t1_sample = data_path + '/subjects/sample/mri/brain.mgz'
+t1_fsaverage = data_path + '/subjects/fsaverage/mri/brain.mgz'
+
+# compute LCMV beamformer inverse example
+img, stc, src = compute_lcmv_example_data(data_path)
+
+###############################################################################
+# Compute Morph Matrix and Dispersion Volume
+morph = mne.source_estimate.compute_morph_sdr(
+    t1_sample,
+    t1_fsaverage,
+    voxel_size=voxel_size)
+
+write_morph('SDR-sample-fsaverage', morph)
+
+###############################################################################
+# Morph Pre - Computed
+
+morph_precomputed = read_morph('SDR-sample-fsaverage')
+
+stc_vol_to_preC = stc.morph_precomputed('sample', 'fsaverage', src,
+                                        morph_precomputed,
+                                        as_volume=False)
+
+img_vol_to_preC = stc.morph_precomputed('sample', 'fsaverage', src,
+                                        morph_precomputed,
+                                        as_volume=True)
+
+###############################################################################
+# Morph
+stc_vol_to_drct = stc.morph('sample',
+                            'fsaverage',
+                            t1_sample,
+                            t1_fsaverage,
+                            src,
+                            voxel_size=voxel_size,
+                            as_volume=False)
+
+img_vol_to_drct = stc.morph('sample',
+                            'fsaverage',
+                            t1_sample,
+                            t1_fsaverage,
+                            src,
+                            voxel_size=voxel_size,
+                            as_volume=True)
+
+###############################################################################
+# Plot results
+
+# load fsaverage brain (Static)
+t1_s_img = nib.load(t1_fsaverage)
+
+# reslice Static
+t1_s_img_res, t1_s_img_res_affine = reslice.reslice(
+    t1_s_img.get_data(),
+    t1_s_img.affine,
+    t1_s_img.header.get_zooms()[:3],
+    voxel_size)
+
+t1_s_img_res = nib.Nifti1Image(t1_s_img_res, t1_s_img_res_affine)
+
+# select image overlay
+imgs = [index_img(img_vol_to_preC, 0), index_img(img_vol_to_drct, 0)]
+
+# select anatomical background images
+t1_imgs = [t1_s_img_res, t1_s_img_res]
+
+# slices to show for Static volume
+slices_s = (0, 0, 0)
+
+slices = [slices_s, slices_s]
```

Comment:
All of this slice stuff is more than we want to ask users to understand. Can you simplify it? Does the `as_volume=True` in the morph help?

Reply:
This one is a bit tricky, since the output of the volumetric morph, in case it is stc is not compatible with src anymore... Which means that ``stc.as_volume(src)`` does not give the correct space... So I thought for now maybe it's good if the user is able to choose between a Nifti and a VolSourceEstimate (that cannot be used with src)

Reply:
Thinking ahead, maybe we could make it so they could they do `vol_stc.as_volume(morpher)` if the `vol_src` is the result of a morph. Perhaps we can design `morpher` to have enough information to do this. Then the users can supply `src` or `morpher` as the first arg, depending on their pipeline

Reply:
In other words, if you want NIfTI you do `orig_vol_stc.morph(morpher).as_volume(morpher)`